src/components/ui/nav/index.tsx

```
@@ -240,19 +298,58 @@ export const Nav = () => {
       </div>
 
       <div className="flex items-center justify-end py-2 lg:pt-3 lg:pr-3 w-full gap-2">
+        {shouldShowUsageIndicator && usageThreshold && (
+          <Tooltip content={`Usage: ${Math.round(maxUsagePercentage)}%`}>
+            <button
+              type="button"
+              role="button"
+              aria-label={`Usage at ${Math.round(maxUsagePercentage)}% - Click to upgrade plan`}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)